src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySourceConfig.java

```diff
@@ -66,6 +66,16 @@
     + "project ID.")
   private String datasetProject;
 
+  @Macro
+  @Nullable
+  @Description("Partition start date. This value is ignored if the table does not support partitions.")
```

Comment:
does not support partitions -> does not support partitioning.

Reply:
Done.